docs/pages/admin-guides/management/security/selinux.mdx

```diff
@@ -0,0 +1,198 @@
+---
+title: "SELinux Module for Teleport SSH"
+description: "How to configure SELinux to enforce Teleport SSH."
+---
+
+<Admonition type="note">
+
+This feature is currently a preview feature and is in active development. It will be stable
+in a future release of Teleport.
+
+</Admonition>
+
```

Comment:
We should include an introductory paragraph that lays out the purpose and scope of this guide.